common/lib/aws_client.ts

```diff
@@ -58,6 +58,8 @@ export abstract class AwsClient extends EventEmitter {
 
     this.properties = new Map<string, any>(Object.entries(config));
 
+    driverDialect.setupInitialProperties(this.properties);
```

Comment:
I think it's better to setup these properties in connection provider connect() method.

Reply:
or in driver dialect connect()